### PR TITLE
Fix time complexity of usaco-760.mdx

### PR DESCRIPTION
**Pull request by theMasix**
the `find` method has a for O(n) in it. and `find` function executes in another for loop. So the final time complexity is O(n^2) - Checkout line `83` and `56` of the file

_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
hold on- the c++ sol is n^2 but the python solution is just n?

**Comment by nathangong** [comment]
i think we should optimize the C++ one then

**Comment by theMasix** [comment]
If we wanna optimize the c++ solution, the easiest way is creating a
`revSwapOrder` which holds the orders in reversed. Then each query to get
the origin of the swap executes in O(1). And the total time complexity will
be O(n).

**Review by dutinmeow — APPROVED** [review]
If it isn't broken then there's no need to fix the code

File: solutions/usaco-760.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ If the $i$th cow moves to $a_i$ after one shuffle, then the cow at $a_i$ was at
 
 ## Implementation
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-760.mdx:13]:
```suggestion
## Implementation
```

File: solutions/usaco-760.mdx [hunk 2]
```diff
@@ -11,7 +11,7 @@ If the $i$th cow moves to $a_i$ after one shuffle, then the cow at $a_i$ was at
...
 
-**Time Complexity:** $\mathcal{O}(N)$
+**Time Complexity:** $\mathcal{O}(N^2)$
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-760.mdx:14]:
```suggestion
**Time Complexity:** $\mathcal{O}(N^2)$

```

**Review by Aadit-Ambadkar — CHANGES_REQUESTED** [review]
spacing

File: solutions/usaco-760.mdx (hunk 1 above)
Comment (dutinmeow) [reply solutions/usaco-760.mdx:13]: wait what was changed

**Pull request closed without merging**